src/plugins/index_pattern_field_editor/public/components/field_format_editor/editors/url/url.tsx

```diff
@@ -59,9 +58,6 @@ export class UrlFormatEditor extends DefaultFormatEditor<
 > {
   static contextType = contextType;
   static formatId = 'url';
-  // TODO: @kbn/optimizer can't compile this
```

Comment:
Also looks like URL formatter image previews are broken: 
![Screenshot 2021-02-11 at 21 22 46](https://user-images.githubusercontent.com/7784120/107694115-4a815a80-6caf-11eb-99c3-7fb00b8d40b6.png)

Probably because assets are not moved